app.js

```diff
@@ -0,0 +1,50 @@
+import data from './data.js';
+
+const form = document.querySelector('#book-form');
+const bookTitle = document.querySelector('#title');
+const bookAuthor = document.querySelector('#author');
+const bookSection = document.querySelector('.books');
+
+const addBook = (author, title) => {
+  data.push({ author, title });
+  return data.length - 1;
+};
+
+const displayBook = (author, title, indx) => {
+  const bookCard = `
+    <div class="book-card">
+      <h2 class="book">${title}</h2>
+      <h3 class="author">${author}</h3>
+      <button class="btn-remove" id="btn-rem-${indx}" data-index="${indx}" type="button">Remove</button>
+      <hr />
+    </div>
+  `;
+  bookSection.insertAdjacentHTML('beforeend', bookCard);
+  return `btn-rem-${indx}`;
+};
```

Comment:
Great job, all functionalities work as expected. 
- [ ] Let's also make sure we add a function called for example displayAllBooks what it does just load data from localStorage on page load we could use this eventlistener 
 document.addEventListener('DOMContentLoaded', displayAllBooks);
Check this if you want to know more about [DOMContentLoaded event](https://developer.mozilla.org/en-US/docs/Web/API/Window/DOMContentLoaded_event)

Reply:
Dear @Amrhub, I appreciate your feedback, but please let me know in what part of the requirements is this something we MUST do in order to get the project approved --> here are the requirements [requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m1_plain_js_objects_v1_1.md#project-requirements).
Maybe this could have just been a suggestion?
We will still make this functionality but will appreciate it if you could take it into consideration for your next reviews.

Reply:
@hsztan It is not crystal clear in the requirement, but, the main reason for us to use LocalStorage is to persist the data even if the user leaves/reloads our website. If we don't display it then as a `user` you didn't persist any data, correct?
I totally understand you are correct when I read it, it was not crystal clear but just a default behavior of saving something in local storage is to display on each load or do the process on it.
![image](https://user-images.githubusercontent.com/87689547/158309339-e65c81e2-903d-47f4-a25d-b3414f9e064b.png)

Reply:
@Amrhub, thanks for your feedback 👍. Changes have been made per your request.
I also believe that the purpose of persisting the data in local storage is to recreate back the elements in the DOM on reload.
Thanks for your input 😎.